### PR TITLE
[#4] Moves web_app to web_client, creates cjl script for docker managment

### PR DESCRIPTION
**Pull request by lancelafontaine**
[#4]

- [x] Renames the `web_app` directory to `web_client` to be more explicit
- [x] Creates `cjl`, a bash script to manage everything related to docker for all services
- [x] Configures continuous integration to adapt to above changes

### CODE REVIEW

File: cjl [hunk 1]
```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env bash
```
Comment (arekmano) [inline cjl:1]: lawyer :'(

Comment (arekmano) [reply cjl:1]: Also, I'm worried that a script like this will be unnecessary maintenance

File: .dockerignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+./**/node_modules/
```
Comment (arekmano) [inline .dockerignore]: disagree. each micro-service (docker image) should have its own .dockerignore. Shouldn't be root's problem

File: src/web_client/Dockerfile [hunk 3]
```diff
@@ -4,10 +4,11 @@ WORKDIR /usr/src/app
 
 RUN apt-get install git -y
 
-COPY . .
-
+COPY src/web_client/package.json .
```
Comment (arekmano) [inline src/web_client/Dockerfile]: this will prevent us from being able to build the web_client from its directory

**Review by arekmano — CHANGES_REQUESTED** [review]

File: cjl (hunk 1 above)
Comment (lancelafontaine) [reply cjl:1]: That's a possibility. If we find it to be too cumbersome in the future, I guess we can scrap it.

Now that it works though, adding new functionality should be very trivial. Should also be a huge time saver for most of us.

File: .dockerignore (hunk 2 above)
Comment (lancelafontaine) [reply .dockerignore]: Fixed

File: src/web_client/Dockerfile (hunk 3 above)
Comment (lancelafontaine) [reply src/web_client/Dockerfile]: Fixed

**Comment by arekmano** [comment]
looks good but I absolutely thing that this will be a maintenance issue further down the line.

**Review by arekmano — APPROVED** [review]

**Pull request merged**